src/measure_theory/integral/circle_integral_transform.lean

```diff
@@ -0,0 +1,220 @@
+/-
+Copyright (c) 2022 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+import analysis.complex.cauchy_integral
+import analysis.analytic.basic
+import analysis.calculus.parametric_interval_integral
+import data.complex.basic
+import measure_theory.integral.circle_integral
+/-!
+# Circle integral transform
+
+In this file we define the circle integral transform of a function `f` with complex domain. This is
+defined as $(2πi)^{-1}\frac{f(x)}{x-w}$ where `x` moves along a circle. We then prove some basic
+facts about these functions.
+
+These functions are useful for proving that the uniform limit of a sequence of holomorphic functions
+is holomorphic.
+
+-/
+
+open topological_space set measure_theory interval_integral metric filter function complex
+open_locale interval real nnreal ennreal topological_space big_operators
+
+noncomputable theory
+
+universes u v
+
+variables {E : Type u} [normed_group E] [normed_space ℂ E]
+
+namespace complex
+
+/--Given a function `f : ℂ → E`, this gives the function  $(2πi)^{-1}\frac{f(x)}{x-w}$ where `x`
```

Comment:
Have you spilled coffee on your keyboard? The spacebar seems prone to sticking :-). Please search-and-replace these out, and also any instances of extra spaces inside brackets, `( foo )` instead of `(foo)`; somebody else will nitpick this if I don't.

Reply:
Ha yes, or fat thumbs! I think I got them all now (now I'll go do the same in all my other PR's...)